vignettes/quick-usage-guide-for-teal.Rmd

```diff
@@ -0,0 +1,80 @@
+---
+title: "Quick usage guide for teal"
```

Comment:
I'm not a fan of this vignette to be created only to show how to modify footer, header and title. Name is competing with "Getting started with teal" and it doesn't say exactly what this vignette is about. Vignette is more about "Modifying teal app elements", but the control we provide is so small that it rather shouldn't be yet published.

Reply:
Okay, makes sense. I will remove it. We can circle back on vignettes later.